build.xml

```diff
@@ -221,7 +214,15 @@
       <arg value="--always"/>
     </exec>
     <!-- Set version if git describe failed. -->
-    <property name="adaptorlib.version" value="unknown"/>
+    <property name="adaptorlib.version" value="unknown-${DSTAMP}"/>
```

Comment:
Yeah, so version was "unknown" and the suffix was "$time", and now both are "unknown-$time".